package.json

```diff
@@ -7,27 +7,39 @@
   "homepage": "https://github.com/LLK/scratch-vm#readme",
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@github.com/LLK/scratch-vm.git"
+    "url": "https://github.com/LLK/scratch-vm.git"
```

Comment:
?

Reply:
I'm wondering if you could still use SSH here and pass the credentials as the bot user - similar to the way we are doing commits with SJR and GUI (I think).